test/integration/features/profile.feature

```diff
@@ -2,104 +2,176 @@
+
+  Scenario: As user, I can get status of active profile 'foo'
+    Given Profile "foo" of Minishift has state "Running"
+     When executing "minishift status" succeeds
+     Then stdout should match "Minishift:\s*Running\nProfile:\s*foo\nOpenShift:\s*Running"
+
+  Scenario: As user, I can set 'minishift' as the active profile
+    Given Profile "foo" of Minishift has state "Running"
+     When executing "minishift profile set minishift" succeeds
+     Then stdout should contain
       """
       Profile 'minishift' set as active profile
       """
 
-Scenario: User should be able to list 'minishift' as the active profile
-    When executing "minishift profile list" succeeds
-    Then stdout should contain
-    """
-    - minishift	Running		(Active)
-    """
-
-Scenario: User should be able to delete profile 'foo'
-    When executing "minishift profile delete foo --force" succeeds
-    Then stdout should contain
-    """
-    Profile 'foo' deleted successfully
-    """
-
-Scenario: User can not delete default profile 'minishift'
-    When executing "minishift profile delete minishift --force"
-    Then exitcode should equal "1"
-     And stderr should contain
-     """
-     Default profile 'minishift' can not be deleted
-     """
+  Scenario: As user, I can list 'minishift' as the active profile
+    Given Profile "minishift" of Minishift has state "Stopped"
+     When executing "minishift profile list" succeeds
+     Then stdout should match "-\s*foo\s*Running"
+      And stdout should match "-\s*minishift\s*Stopped\s*\(Active\)"
+
+  Scenario: As user, I can use profile alias command 'instance' to list 'minishift' as the active profile
+    This alias instance of profile can be applied to all the commands instead of profile.
+    Here only list is demonstrated with this command.
+    Given Profile "minishift" of Minishift has state "Stopped"
+     When executing "minishift instance list" succeeds
+     Then stdout should match "-\s*foo\s*Running"
+      And stdout should match "-\s*minishift\s*Stopped\s*\(Active\)"
+
+  Scenario: As user, I can use profile alias command 'profiles' to list 'minishift' as the active profile
+    This alias 'profiles' of profile can be applied to all the commands instead of profile.
+    Here only list is demonstrated with this command.
+    Given Profile "minishift" of Minishift has state "Stopped"
+     When executing "minishift profiles list" succeeds
+     Then stdout should match "-\s*foo\s*Running"
+      And stdout should match "-\s*minishift\s*Stopped\s*\(Active\)"
+
+  Scenario: As user, I can delete profile 'foo'
+    Given Profile "minishift" of Minishift has state "Stopped"
+     When executing "minishift profile delete foo --force" succeeds
+     Then stdout should contain
+      """
+      Profile 'foo' deleted successfully
+      """
+
+  Scenario: As user, I can't delete default profile 'minishift'
```

Comment:
Can one delete absolutely non-existing profile? +1 for adding this scenario

Reply:
Nice to have this scenario